ctapipe/io/hessiofilereader.py

```diff
@@ -0,0 +1,211 @@
+from astropy import units as u
+from astropy.coordinates import Angle
+from astropy.time import Time
+from ctapipe.core import Provenance
+from ctapipe.io.eventfilereader import EventFileReader
+from ctapipe.io.containers import DataContainer
+from ctapipe.instrument import TelescopeDescription, SubarrayDescription
+
+
+COUNT = 0
+
+
+class HessioFileReader(EventFileReader):
```

Comment:
Needs a Docstring

Reply:
I think `EventFileReader` should support `with EventFileReader() as reader`. The HessioReader should then call `close_hessio` in the `__exit__` method.